From 6876948f7e74432b2f13177ed6f7c0dc524bae5a Mon Sep 17 00:00:00 2001
From: kim <kyoungbin1996@gmail.com>
Date: Fri, 13 Sep 2024 12:05:49 +0900
Subject: [PATCH 1/2] =?UTF-8?q?ffix:=20segTree=20=EA=B3=B5=EB=B0=B1?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

---
 segTree.py | 2 --
 1 file changed, 2 deletions(-)

diff --git a/segTree.py b/segTree.py
index a07140e..8665343 100644
--- a/segTree.py
+++ b/segTree.py
@@ -21,7 +21,6 @@ def find(node, start, end, left, right) :
     
     if left > end or right < start :
         return 0
- 
 
     if left <= start and end <= right :
         return tree[node]
@@ -52,5 +51,4 @@ def update2(node, index):
         ## update 로직
         tree[node] = tree[node*2] + tree[node*2+1]
 
-
 print(tree)
\ No newline at end of file
-- 
2.36.1

